README.md

```diff
@@ -90,5 +85,3 @@ ios_application(
 )
 ```
 
-See the [examples](https://github.com/ob/rules_ios/tree/master/examples)
```

Comment:
why is this stuff removed from the README?

Reply:
404, no examples in master branc